### PR TITLE
add danish translations to some packages

### PR DESCRIPTION
**Pull request by rlvk-vk**
### What does it do?

I've simply added danish translations for some of the default packages.

### Why is it needed?

I want to use Strapi for some projects and I live in Denmark, so it is quite useful that I can provide the backend in native language

### Related issue(s)/PR(s)

It is not to my knowledge.

### CODE REVIEW

**Comment by soupette** [comment]
@rlvk-vk thanks a lot for your contribution ! Can you make sure the DCO checks so we can review your PR?

**Comment by rlvk-vk** [comment]
> @rlvk-vk thanks a lot for your contribution ! Can you make sure the DCO checks so we can review your PR?

This is the first time I'm contributing, so I'm not quite sure what that means 😊 sorry

**Comment by petersg83** [comment]
Hi :)
It means to signoff when you commit:
```
git commit -m "add danish translation" --signoff
```

Now that you already have several commits, I advice to do the following:
```bash
git pull
git reset HEAD~1 --hard # to remove the merge commit
git commit --amend --signoff
git push -f
```

**Comment by rlvk-vk** [comment]
Thank you for your reply @petersg83 but I am simply getting a 403 remote: Permission to strapi/strapi.git denied to rlvk-vk, when trying to do so.

**Comment by soupette** [comment]
@rlvk-vk can you make sure to accept our CLA so we can merge your PR?

**Comment by rlvk-vk** [comment]
I’ve signed the CLA now.

Med venlig hilsen
Robin Louis van Komen

**Comment by soupette** [comment]
> I’ve signed the CLA now. Med venlig hilsen Robin Louis van Komen
> […](#)
> On 1 Dec 2020, 09.16 +0100, cyril lopez ***@***.***>, wrote: @rlvk-vk can you make sure to accept our CLA so we can merge your PR? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

Thanks a lot!

It appears your PR contains some conflicts do you mind checking them?

**Comment by rlvk-vk** [comment]
> > I’ve signed the CLA now. Med venlig hilsen Robin Louis van Komen
> > […](#)
> > On 1 Dec 2020, 09.16 +0100, cyril lopez _**@**_.***>, wrote: @rlvk-vk can you make sure to accept our CLA so we can merge your PR? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> Thanks a lot!
> 
> It appears your PR contains some conflicts do you mind checking them?

I'm not quite sure where I can see these conflicts?

**Comment by soupette** [comment]
> > > I’ve signed the CLA now. Med venlig hilsen Robin Louis van Komen
> > > […](#)
> > > On 1 Dec 2020, 09.16 +0100, cyril lopez _**@**_.***>, wrote: @rlvk-vk can you make sure to accept our CLA so we can merge your PR? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.
> > 
> > 
> > Thanks a lot!
> > It appears your PR contains some conflicts do you mind checking them?
> 
> I'm not quite sure where I can see these conflicts?

If you try to update your branch you'll see some conflicts in this file : 
packages/strapi-plugin-upload/admin/src/translations/index.js

**Comment by alexandrebodin** [comment]
Hi @rlvk-vk will you be able to fix the conflicts ? let us know if you need some help ;)

**Comment by rlvk-vk** [comment]
I have fixed the merge conflict now!

File: packages/strapi-plugin-email/admin/src/translations/dk.json
Comment (soupette) [inline packages/strapi-plugin-email/admin/src/translations/dk.json:2]: Can you also translate this file?

**Review by soupette — CHANGES_REQUESTED** [review]

File: packages/strapi-plugin-email/admin/src/translations/dk.json
Comment (rlvk-vk) [reply packages/strapi-plugin-email/admin/src/translations/dk.json:2]: The file is already translated. It’s just the same in danish as in english

**Review by soupette — APPROVED** [review]
LGTM

**Comment by alexandrebodin** [comment]
Hi @rlvk-vk FYI there is a unit test not passing in the front. Will wait for the fix before merging ;)

**Comment by alexandrebodin** [comment]
Hi @rlvk-vk did you have time to check the tests ?

**Review by madsh93 — COMMENTED** [review]
Translation looks solid. Another danish guy approves :-)

**Comment by alexandrebodin** [comment]
Sorry right now we cannot merge it because the tests are failing on the PR :) it will need to be updated first ;)

**Comment by rlvk-vk** [comment]
Sorry, I didn't realize that I was supposed to do anything. I Have made the adjustments, to make the tests succeed!

However I'm not quite sure about the codecov tests that are failing, should I do anything about this? It shouldn't really concern the feature that I'm commiting.

**Review by alexandrebodin — APPROVED** [review]
LGTM, Awesome job 💯

**Pull request merged**
